CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- Add `disableUnavailableSelectOptions` prop for `SKUSelector` in order to disable unavailable items for `displayMode` select
```

Comment:
Just to make it easier, when you add an entry to the CHANGELOG, add it right below the `[Unreleased]` header, without an empty line. This will avoid conflicts :)

```suggestion
## [Unreleased]
### Added
- `disableUnavailableSelectOptions` prop to `SKUSelector` in order to disable unavailable items when `displayMode` is set to `select`.
```